.github/ISSUE_TEMPLATE/BUG_REPORT.yml

```diff
@@ -0,0 +1,57 @@
+name: Bug report
+description: Help us improve InstantSearch
+labels:
+  - triage
+body:
+- type: textarea
+  attributes:
+    label: 🐛 Bug description
+    description: A clear and concise description of what the bug is.
+  validations:
+    required: true
+- type: textarea
+  attributes:
+    label: 🔍 Bug reproduction
+    description: Steps to reproduce the behavior.
+    placeholder: |
+      1. Go to ...
+      2. Click on ...
+      3. Scroll down to ...
+      4. See error
+
+      You can attach screenshots by dragging & dropping, selecting or pasting them on the textarea.
+  validations:
+    required: true
+- type: input
+  attributes:
+    label: Live reproduction
+    description: |
+      A live example helps a lot! Fork the sandbox relevant to your package, reproduce the bug and paste the URL here.
+      Sandboxes: [InstantSearch.js](https://codesandbox.io/s/github/algolia/create-instantsearch-app/tree/templates/instantsearch.js)
+    placeholder: https://codesandbox.io/...
+  validations:
+    required: true
+- type: textarea
+  attributes:
+    label: 💭 Expected behavior
+    description: A clear and concise description of what you expected to happen.
+  validations:
+    required: true
+- type: textarea
+  attributes:
+    label: Environment
+    placeholder: |
+      - OS: [e.g. Windows / Linux / macOS / iOS / Android]
+      - Browser: [e.g. Chrome, Safari]
+      - InstantSearch version: [e.g. 4.49.2]
```

Comment:
I've seen examples of dedicated dropdowns for OS / Browser selection, but I'm not sure it brings a lot of value and would make us keep the lists up to date. So I kept the textarea instead. Let me know what you think.